Library/Formula/luvi.rb

```diff
@@ -0,0 +1,19 @@
+class Luvi < Formula
+  desc "A project in-between luv and luvit, lua packages."
+  homepage "https://github.com/luvit/luvi"
+  url "https://github.com/luvit/luvi/releases/download/v2.1.8/luvi-src-v2.1.8.tar.gz"
+  sha256 "1ef449c6a76bcf7a03c8bbcdd5423b0366d63b10c52fd463ca2ba8aaf9248781"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make", "regular-asm"
+    system "make"
+    bin.install "build/luvi"
+  end
+
+  test do
+    system "make", "test"
```

Comment:
So the problem is it will build the main binary twice?  If so, I can just run the contents of the test target directly using `bin/luvi`.